log4j-core/src/main/java/org/apache/logging/log4j/core/appender/rolling/CompositeTriggeringPolicy.java

```diff
@@ -50,6 +50,7 @@ private CompositeTriggeringPolicy(final TriggeringPolicy... triggeringPolicies)
     @Override
     public void initialize(final RollingFileManager manager) {
         for (final TriggeringPolicy triggeringPolicy : triggeringPolicies) {
+            LOGGER.debug("Initializing triggering policy ", triggeringPolicy.toString());
```

Comment:
nit: I don't think toString isn't necessary here, the logger will get the string value if debug is enabled